fpga_interchange/main.cc

```diff
@@ -55,6 +55,8 @@ po::options_description FpgaInterchangeCommandHandler::getArchOptions()
     specific.add_options()("netlist", po::value<std::string>(), "FPGA interchange logical netlist to read");
     specific.add_options()("phys", po::value<std::string>(), "FPGA interchange Physical netlist to write");
     specific.add_options()("package", po::value<std::string>(), "Package to use");
+    specific.add_options()("rebuild_lookahead", "Ignore lookahead cache and rebuild");
```

Comment:
nextpnr convention is to use `-`, not `_`, in multi-word arguments

Reply:
Fixed